### PR TITLE
Change default keyboard shortcuts

### LINKED ISSUE #41207: Change default shortcuts on Windows and Linux
The default shortcuts on those platforms conflict with some of the default bash and nano shortcuts (e.g. Ctrl + E, Ctrl + K). This makes using a terminal really painful. We had a bunch of people complain about this through the community Slack and the feedback form. I'm surprised we don't hear about this more often.

On top of this, for some reason some keyboard interactions don’t work in Connect but do work in the Web UI terminal, e.g. cycling backwards the suggestion list in zsh. (#28903)

Before a major release, we should take a look at Windows Terminal, VSCode and some popular Linux terminals to see how they handle terminal shortcuts and do the same in Connect. For example, I know that most shortcuts in Windows Terminal use Ctrl + Shift, not just Ctrl alone.

Originally, we had the idea of forwarding shortcuts to the terminal somehow (#23104), but it might be easier to just change the defaults.

### PR DESCRIPTION
**Pull request by gzdunek**
Closes https://github.com/gravitational/teleport/issues/41207.

I've looked at vscode, tabby, Windows Terminal and the Ubuntu terminal and I see that they usually use `Ctrl+Shift+*` or `Ctrl+Alt+*` for shortcuts. I decided to change ours to `Ctrl+Shift+*` since they are more popular. I also looked at the [bash](https://gist.github.com/tuxfight3r/60051ac67c5f0445efee) and [nano](https://www.nano-editor.org/dist/latest/cheatsheet.html) shortcuts and we shouldn't conflict with them now.

I was thinking about changing the `Ctrl+1 - Ctrl+9` shortcuts on Windows to `Ctrl+Shift+1 - Ctrl+Shift+9` to match Windows Terminal, but they don't cause conflicts, so I left them intact.

### CODE REVIEW

File: web/packages/teleterm/src/services/config/appConfigSchema.ts [hunk 1]
```diff
@@ -161,15 +161,15 @@ const getDefaultKeymap = (
         tab7: 'Ctrl+7',
         tab8: 'Ctrl+8',
         tab9: 'Ctrl+9',
-        closeTab: 'Ctrl+W',
-        newTab: 'Ctrl+T',
-        newTerminalTab: 'Ctrl+Shift+T',
+        closeTab: 'Ctrl+Shift+W',
+        newTab: 'Ctrl+Shift+T',
+        newTerminalTab: 'Ctrl+Shift+`',
```
Comment (gzdunek) [inline web/packages/teleterm/src/services/config/appConfigSchema.ts:166]: This shortcut matches vscode.

**Review by avatus — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]
Can we remove the changelog note from the PR description and write 2-3 sentences in CHANGELOG.md under breaking changes instead?

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +34,13 @@ more performant, `tbot ssh-proxy-command`. See the
 [v16 upgrade guide](docs/pages/machine-id/reference/v16-upgrade-guide.mdx) for
 more details.
 
+#### Default keyboard shortcuts in Teleport Connect have been changed
+
+Some of the default shortcuts conflicted with the default bash or nano shortcuts
```
Comment (ravicious) [inline CHANGELOG.md:39]:
```suggestion
On Windows and Linux, some of the default shortcuts conflicted with the default bash or nano shortcuts 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -34,6 +34,13 @@ more performant, `tbot ssh-proxy-command`. See the
 [v16 upgrade guide](docs/pages/machine-id/reference/v16-upgrade-guide.mdx) for
 more details.
 
+#### Default keyboard shortcuts in Teleport Connect have been changed
+
+Some of the default shortcuts conflicted with the default bash or nano shortcuts 
+(e.g. Ctrl + E, Ctrl + K).
+These have been changed to a combination of Ctrl + Shift + *, see [configuration](docs/pages/connect-your-client/teleport-connect.mdx/#configuration)
```
Comment (ravicious) [inline CHANGELOG.md:41]:
```suggestion
On those platforms, the default shortcuts have been changed to a combination of Ctrl + Shift + *, see [configuration](docs/pages/connect-your-client/teleport-connect.mdx/#configuration)
```

Let's also add a note about the default shortcut for a new terminal on macOS.

File: docs/pages/connect-your-client/teleport-connect.mdx [hunk 4]
```diff
@@ -67,7 +67,7 @@ which cluster the current tab is bound to, and the **Share Feedback** button in
 ## Connecting to an SSH server
 
 1. Open a tab with cluster resources by clicking on the plus symbol at the right end of the tab bar.
-   You can also press <span style="white-space: nowrap;">`Ctrl/Cmd + T`</span> to achieve the same result.
+   You can also press <span style="white-space: nowrap;">`Ctrl+Shift+T/Cmd+T`</span> to achieve the same result.
```
Comment (ravicious) [inline docs/pages/connect-your-client/teleport-connect.mdx:70]: Now that you've removed the spaces, those spans can be removed.

**Review by ravicious — APPROVED** [review]
Good job, I'll give these a try during the test plan.

**Pull request merged**